src/dto/Password.java

```diff
@@ -0,0 +1,20 @@
+package dto;
+
+/**
+ * Created by freya on 17-02-2017.
+ */
+public class Password {
+    private String password;
+
+    public Password() {
+        //TODO: Generate password and save in variable password
+    }
+
+    public String getPassword() {
```

Comment:
En getter for et password virker som en mindre god idé. Nok hellere en `checkPassword` metode, som returnerer en boolean og sammenligner nogle passwords.